apps/renterd/contexts/config/deriveAllowance.ts

```diff
@@ -50,17 +50,17 @@ export function derivePricingFromAllowance({
   downloadWeight = 5,
   uploadWeight = 1,
 }: {
-  allowanceMonth: BigNumber
+  allowanceMonth?: BigNumber
   allowanceFactor?: number
-  storageTB: BigNumber
-  downloadTBMonth: BigNumber
-  uploadTBMonth: BigNumber
-  redundancyMultiplier: BigNumber
+  storageTB?: BigNumber
+  downloadTBMonth?: BigNumber
+  uploadTBMonth?: BigNumber
+  redundancyMultiplier?: BigNumber
   storageWeight?: number
   downloadWeight?: number
   uploadWeight?: number
 }) {
-  // Return null if zero values are provided.
+  // Return undefined if zero values are provided.
```

Comment:
`nit` would be easier on the eyes if we would assert they're zero instead of not greater than because they can't be negative right